### PR TITLE
Add company page

### PR DESCRIPTION
**Pull request by maartenvanvliet**
This PR adds a company page as mentioned in https://github.com/beam-community/elixir-companies/issues/533

It is very basic for now, just a page-wide version of the company card.

### CODE REVIEW

File: lib/companies/companies.ex [hunk 1]
```diff
@@ -76,7 +76,13 @@ defmodule Companies.Companies do
   ** (Ecto.NoResultsError)
 
   """
-  def get!(id), do: Repo.get!(Company, id)
+  def get!(id, opts \\ []) do
```
Comment (gemantzu) [inline lib/companies/companies.ex]: TIL, thank you!

**Comment by gemantzu** [comment]
I don't really like how the show page looks. any thoughts @doomspork ?

File: lib/companies/companies.ex (hunk 1 above)
Comment (doomspork) [reply lib/companies/companies.ex]: Nice! @maartenvanvliet great minds think alike! I just implemented this pattern in another project.

It’s also possible, but more work, to work this into a query builder such that you don’t need _N+1_ queries. 

Really like this 👌

**Comment by maartenvanvliet** [comment]
Screenshot 2019-10-28 at 11 25 24

In this PR it looks like this, so the card that was already used but then wider. It could be made a little smaller on desktop, and perhaps not use the card. Any other suggestions?

**Comment by gemantzu** [comment]
I think we should consider moving away from the cards, at least for the company show page (maybe for the index as well, idk).

**Comment by maartenvanvliet** [comment]
Screenshot 2019-10-29 at 15 56 22

This is what I came up with by removing the card :)

**Comment by gemantzu** [comment]
That's already much better. @doomspork ?

**Comment by doomspork** [comment]
Sorry @maartenvanvliet I have been very busy the last few weeks. I think this looks great. Could you rebase your PR so we can merge?

**Comment by maartenvanvliet** [comment]
Done :)

**Comment by gemantzu** [comment]
Thank you @maartenvanvliet !!!!

**Pull request merged**